onnx/defs/schema.h

```diff
@@ -558,6 +560,14 @@ class OpSchema final {
     return tensor_inference_function_ ? true : false;
   }
 
+  bool has_function() const {
```

Comment:
HasFunction? I'm seeing the other two functions you added with name "FunctionBody" and "GetFunction". Let's keep the name consistent.